### DISCUSSION TITLE
one form in different components

### DISCUSSION THREAD

**Discussion by Ladvace** [ant-design/ant-design#48335] [Q&A]
is it possible to split a form between different "components"? I basically have a page where you can move back and forward between different steps and I would like to split different fields of the same form in different steps.
I tried doing it but at the end I just get fields of the last "step"

**Comment by Wxh16144** [comment]
Like this https://procomponents.ant.design/components/steps-form ?

**Reply by Ladvace** [reply]
Yes! is this a premium/paid version of antd?

**Comment by Ladvace** [comment]
is also possible to remove this from the stops form?

<img width="825" alt="image" src="https://github.com/ant-design/ant-design/assets/31326058/e80bd363-2281-4312-87a8-d42968086f6e">
